src/bootstrap/bootstrap.ts

```diff
@@ -0,0 +1,23 @@
+import 'reflect-metadata';
+import express from 'express';
+import { appConfig } from '../config';
+import { ErrorHandler, logRequestMiddleware, privateRoutes, rateLimiter } from '../middlewares';
+import { taskRouter, userRouter } from '../modules/router';
+import { logRoutes } from './log-routes';
+
+export const bootstrap = () => {
+  const server = express();
+  const port = appConfig.port;
+
+  server.use(express.json());
+  server.use(privateRoutes);
+  server.use(logRequestMiddleware);
+  server.use(rateLimiter);
+  server.use('/user', userRouter);
+  server.use('/task', taskRouter);
+  server.use(ErrorHandler);
+
+  logRoutes(server);
+
+  server.listen(port, () => console.log(`Server started on port ${port}`));
+};
```

Comment:
```suggestion
};

bootstrap();
```

Тут ты немного перестарался)  
1. Объединить всё в `bootstrap` - правильно.
2. Выносить всё в отдельный файл `bootstrap/bootstrap`, импортировать эту функцию в `main.ts` - излишне.

Достаточно эту функцию `bootstrap` перенести прямо в `main.ts` файл, и там же её и вызвать.  
То есть вот этот код прямо перенеси в `main.ts`, и всё, не нужен доп файл `bootstrap/bootstrap.ts`